src/Microsoft.AspNetCore.ResponseCaching/Streams/ResponseCachingStream.cs

```diff
@@ -183,19 +183,12 @@ public override void WriteByte(byte value)
             }
         }
 
-#if NETSTANDARD1_3
-        public IAsyncResult BeginWrite(byte[] buffer, int offset, int count, AsyncCallback callback, object state)
```

Comment:
Is this right? Wouldn't you want this code and not the other one? This branch was the one used by the `netstandard` build (which was actually the only build anyway)